src/components/v2v3/V2V3Project/ProjectPageRightCol.tsx

```diff
@@ -0,0 +1,30 @@
+import { Col } from 'antd'
+import { NftRewardsSection } from 'components/NftRewards/NftRewardsSection'
+import { PayProjectForm } from 'components/Project/PayProjectForm'
+import ProjectActivity from './ProjectActivity'
+import { COL_SIZE_MD } from './V2V3Project'
+
+export function ProjectPageRightCol({
+  hasNftRewards,
```

Comment:
**note (non-blockiing)** potential tech debt, maybe we can just use existing context in this component rather than prop-drilling.